identity_test.go

```diff
@@ -73,3 +74,207 @@ var _ = Describe("newIdentity", func() {
 		Expect(err).Should(HaveOccurred())
 	})
 })
+
+var _ = Describe("generate", func() {
```

Comment:
Don't you have a `generete_test.go` file to remove then?

**EDIT** Just found that this is done in the wrong commit ("make a Target type and use it in identities"), you should move it here.

Reply:
Right, will change